app/scripts/controllers/buildConfig.js

```diff
@@ -197,6 +203,44 @@ angular.module('openshiftConsole')
           }
         );
 
+        DataService.list("configmaps", context, null, { errorNotification: false }).then(function(configMapData) {
+          configMapDataOrdered = orderByDisplayName(configMapData.by("metadata.name"));
+          $scope.valueFromObjects = configMapDataOrdered.concat(secretDataOrdered);
+        }, function(e) {
+          if (e.code === 403) {
+            return;
+          }
+
+          NotificationsService.addNotification({
+            id: "create-builder-list-config-maps-error",
+            type: "error",
+            message: "Could not load config maps.",
+            details: getErrorDetails(e)
+          });
+        });
+
+        DataService.list("secrets", context, null, { errorNotification: false }).then(function(secretData) {
+          secretDataOrdered = orderByDisplayName(secretData.by("metadata.name"));
+          $scope.valueFromObjects = secretDataOrdered.concat(configMapDataOrdered);
```

Comment:
Let's try to keep a consistent ordering as above so we get the same result regardless of which loads first. (It will show the select groupings in different orders depending on how we concat the arrays.)

```js
$scope.valueFromObjects = configMapDataOrdered.concat(secretDataOrdered);
```